### ISSUE TITLE
Allow providing artifact to upload instead of building from source

### ISSUE THREAD

**Issue by Nimrodda** [Triple-T/gradle-play-publisher#360]
We store our RC builds on a server. When a build reaches QA approval, we'd like to upload that specific build. Our CI executes a job that will run publishApk task on demand on a clean directory. Placing a pre-built APK in the respective `build/outputs` folder doesn't seem to make publishApk task to take the APK and upload it. Instead, it goes through the whole build process from scratch. 
Is it possible to make publishApk task upload existing APK instead of building from source?

**Comment by SUPERCILEX** [comment]
Not right now, but this feature will be available in v2.0.

**Comment by SUPERCILEX** [comment]
Hallelujah people! This will most likely be shipped in a week or two. 🎉 In the meantime, please try a [snapshot build](https://github.com/Triple-T/gradle-play-publisher/blob/master/README.md#snapshot-builds) to make sure the API fits your use case.

Lighting docs:
- You can use the new CLI option: `./gradlew publishApk --artifact-dir path/to/apk/dir`
- Or specify the `artifactDir` property in your Gradle DSL:
  ```kt
  play {
      // ...
      artifactDir = file("path/to/apk/dir")
  }
  ```
